operating/respond-to-violations.adoc

```diff
@@ -0,0 +1,42 @@
+[id="respond-to-violations"]
+= Responding to violations
+include::modules/common-attributes.adoc[]
+:context: respond-to-violations
+
+toc::[]
+
+[role="_abstract"]
+Using {product-title} you can view policy violations, drill down to the actual cause of the violation, and take corrective actions.
+
+{product-title} built-in policies identify a variety of security findings, including vulnerabilities (CVEs), violations of DevOps best practices, high-risk build and deployment practices, and suspicious runtime behaviors.
+Whether you use the default out-of-box security policies or use your own custom policies, {product-title} reports a violation when an enabled policy fails.
+
+include::modules/violations-view.adoc[leveloffset=+1]
+
+[id="view-violation-details"]
+== Viewing violation details
+When you select a violation in the *Violations* view, the *Violation Details* open in a panel on the right.
+
+The *Violation Details* panel shows detailed information grouped by multiple tabs.
+
+include::modules/violation-view-violation-tab.adoc[leveloffset=+2]
+
+[id="using-comments-and-tags"]
+=== Using comments and tags
+You can use tags and comments to specify what is happening with violations to keep your team up to date.
+Comments allow you to add text notes to violations and tags allow you to categorize your violations.
+
+:smallporv: violation
```

Comment:
What are these? I don't see these attributes used anywhere

Reply:
The changes for those attributes are part of https://github.com/openshift/openshift-docs/pull/33962
- PS [my comment](https://github.com/openshift/openshift-docs/pull/33962#pullrequestreview-692470610)

Reply:
Cool, I'll add a comment there, but this is fine then. Thanks!